src/blocks/owc-openkaarten/streetmap/assets/scripts/vue/TheMap.vue

```diff
@@ -491,12 +495,18 @@ const handleSearch = async (query) => {
 	&__controls {
 		position: absolute;
 		inset-block-start: 20px;
-		inset-inline-start: 20px;
+		inset-inline-start: 10px;
 		z-index: 1000;
 		display: flex;
 		gap: 0.5rem;
-		max-width: min(450px, calc(100% - 2rem));
 		width: 100%;
+    @media only screen and (min-width: 768px) {
```

Comment:
inset & max-width :P waarom wordt hier dan niet inline-size of max-inline-size gebruikt?